src/main/java/com/example/housingmanagement/api/mappers/HouseMapper.java

```diff
@@ -5,20 +5,24 @@
 import com.example.housingmanagement.api.responses.HouseResponse;
 import org.springframework.stereotype.Component;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
 
 @Component
 public class HouseMapper implements HouseMapperInterface {
 
     @Override
     public List<HouseResponse> toHouseResponse(List<HouseInternalEntity> allHouseInternalEntities) {
```

Comment:
Jeśli chodzi o nazwy zmiennych (czy to pól, czy parametrów czy zmiennych referencyjnych w środku metody) - to zobacz co podpowiada Ci intellij.
Jak dasz kursor na `allHouseInternalEntities` i wciszniesz ctrl+spacja to Intellij podowie Ci nazwę parametru.
Następnie jak Ci się spodoba to zrób rename (shift + F6) i wtedy ctrl+spacja, a potem enter

Reply:
nice, dobry tip - czasem widzę że sam fajnie podpowiada przy tworzeniu danej zmiennej. Czyli ctrl+spacja wymusza zobaczenie jakie tam podpowiedzi sobie wymyślił.